docs/data/material/components/table/table.md

```diff
@@ -33,7 +33,7 @@ This constraint makes building rich data tables challenging.
 The [`DataGrid` component](/x/react-data-grid/) is designed for use-cases that are focused on handling large amounts of tabular data.
 While it comes with a more rigid structure, in exchange, you gain more powerful features.
 
-{{"demo": "DataTable.js", "bg": "inline"}}
+{{"demo": "DataTable.js", "bg": "outlined"}}
```

Comment:
The rest of the demos in this page use `"bg": true`. Any particular reason why we should use `outlined`? I'm not familiar with when one should be picked over the other.

Reply:
> Any particular reason why we should use outlined?

We don't have specific rules for choosing one over the other. In this case, it was mainly because the DataGrid has a transparent background, and "bg": true would interfere with the demo.

Reply:
👍